data/usp_drug_classification_tidying_script.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python2
+# -*- coding: utf-8 -*-
+"""
+Parse and tidy the USP Drug Classification data from KEGG
+Raw data from http://www.genome.jp/kegg-bin/get_htext?htext=br08302.keg
+
+Created on Sun Feb  5 13:15:04 2017
+
+@author: claire
+"""
+
+import numpy as np
+import pandas as pd
+
+if __name__ == "__main__":
+    
```

Comment:
Is this supposed to be just a script to be executable from the command line? Or should this supply a function that can be run from within a larger `Python` program as well? If it's the former, you don't really need the `if __name__ == "__main__" line. 

I would actually suggest moving the code below into a function `tidy_kegg_data()` or something, and then have it execute that here. This would allow someone to run this from within a larger programme, if necessary.